srcpkgs/Thunar/template

```diff
@@ -11,13 +12,13 @@ configure_args="--enable-gio-unix --enable-gudev
 hostmakedepends="pkg-config intltool"
 makedepends="exo-devel libexif-devel libgudev-devel libnotify-devel xfce4-panel-devel"
 depends="desktop-file-utils hicolor-icon-theme"
-short_desc="Thunar File Manager for XFCE"
-maintainer="Ulf <void@uw.anonaddy.com>"
+short_desc="Thunar File Manager"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
I think these changes can stay?

Reply:
I'll keep old `short_desc`, this package is _the_ Thunar

Reply:
Fair enough, my apologies.